spec/PeriodicBackgroundSync-index.bs

```diff
@@ -145,6 +148,30 @@ Fetch requests within the [=periodicsync event=] while [=periodicsync event/in t
 ## History Leaking ## {#history-leaking}
 Fetch requests within the [=periodicsync event=] while [=periodicsync event/in the background=] may reveal something about the client's navigation history to middleboxes on networks different from the one used to create the [=periodic sync registration=]. For instance, the client might visit site https://example.com, which registers a [=periodicsync event=], but based on the implementation, might not fire until after the user has navigated away from the page and changed networks. Middleboxes on the new network may see the fetch requests that the [=periodicsync event=] makes. The fetch requests are HTTPS so the request contents will not be leaked but the destination of the fetch request and domain may be (via DNS lookups and IP address of the request). To prevent this leakage of browsing history, the user agent MAY choose to only fire [=periodicsync event|periodicsync events=] on the network the [=periodic sync registration=] was made on, with the understanding that it will reduce usability by not allowing synchronization opportunistically.
 
+Security Considerations {#security}
+====================================
+Since Periodic Background Sync is a [=service worker=] based API, [[service-workers#security-considerations]] apply.
+
+In addition, if there is a disparity between the intervals at which the web app can synchronize with its servers and those at which the user visits the site, the [=origin=] gets more power than the user is aware of or intends. This is because the [=origin=] is able to execute code more frequently than the user intends, from within the [=periodicsync event=] while [=periodicsync event/in the background=].
+User agents should therefore implement countermeasures to prevent such disparity.
+
+To mitigate the above, the user agent MUST:
+* Provide UI for users to opt out of firing [=periodicsync events|periodicsync event=] for any given [=origin=].
+* Impose a time limit on the lifetime extension and execution time of a {{PeriodicSyncEvent}} which is stricter than the [=ExtendableEvent/timed out flag|time limit=] imposed for {{ExtendableEvent}}s in general. In particular, any retries of the {{PeriodicSyncEvent}} MAY have a significantly shortened time limit.
```

Comment:
This is redundant with a recommendation in the privacy considerations section. Instead of recommending it twice, find a single place to put the rule, perhaps in https://wicg.github.io/BackgroundSync/spec/PeriodicBackgroundSync-index.html#process-periodic-sync-registrations-algorithm and https://wicg.github.io/BackgroundSync/spec/PeriodicBackgroundSync-index.html#fire-a-periodicsync-event, and then reference it and describe how it helps in the security and privacy considerations sections.

Reply:
done.